packages/runtime/package.json

```diff
@@ -160,6 +160,7 @@
     "react-player": "^1.12.0",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
+    "reselect": "^5.1.1",
```

Comment:
Adding `reselect` for [memoized selector support](https://redux.js.org/usage/deriving-data-selectors#writing-memoized-selectors-with-reselect).